WcaOnRails/app/webpacker/javascript/hooks/useSaveData.js

```diff
@@ -0,0 +1,35 @@
+import { useState, useCallback } from 'react';
+import { fetchJsonOrError } from '../requests/fetchWithAuthenticityToken';
+
+const throwError = (err) => { throw err; };
```

Comment:
Why are we not using a similar thing in the `useLoadedData` hook?

Reply:
I think because I didn't think about it at the time, but that would definitely be something nice to add there!